rpm/0001-Allow-to-use-patch-instead-of-git-to-apply-patches.patch

```diff
@@ -0,0 +1,63 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Bj=C3=B6rn=20Bidar?= <bjorn.bidar@jolla.com>
+Date: Tue, 12 Mar 2024 10:40:59 +0200
+Subject: [PATCH] Allow to use patch instead of git to apply patches
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Signed-off-by: Björn Bidar <bjorn.bidar@jolla.com>
+(cherry picked from commit 1bebaad9ee381dc7c28f9c0da07dcd8bc249ed22)
+---
+ vendor/CMakeLists.txt | 36 +++++++++++++++++++++++++-----------
+ 1 file changed, 25 insertions(+), 11 deletions(-)
+
+diff --git a/vendor/CMakeLists.txt b/vendor/CMakeLists.txt
+index ca4f759869e4d0c791621c88f50942598000eee3..ae8f061a4e95abc23fc500aafdd7bf322422597a 100644
+--- a/vendor/CMakeLists.txt
++++ b/vendor/CMakeLists.txt
+@@ -61,19 +61,33 @@ set(android-vendored
+ 	boringssl)
+ 
+ # XXX: Consider using https://cmake.org/cmake/help/v3.0/module/ExternalProject.html
++
++find_package(Git)
```

Comment:
I think to try to find git is a simpler option, it's a common pattern to find dependencies like this.
Keeping the ability to use git is useful for development.

Reply:
The issue is git might be installed in the build environment (for what ever reason), but I'm not sure it will work if the build is done from sources extracted from a tarball, instead of a git repo.

Reply:
I don't see any reason for that to happen besides local builds where the git repository would be available in any case.